style-guide.md

```diff
@@ -21,51 +21,51 @@ export default () => (
 ✅ DO:
 
 ```js
-// Ejemplo
+// Exemple
 import React from "react"
 export default () => (
-  <div style={{ color: `purple`, fontSize: `72px` }}>Hello Gatsby!</div>
+  <div style={{ color: `purple`, fontSize: `72px` }}>Salut Gatsby!</div>
 )
 ```
 
-✅ ALSO OKAY:
+✅ OK:
 
 ```js
-// Ejemplo
+// Exemple
 import React from "react"
 export default () => (
-  <div style={{ color: `purple`, fontSize: `72px` }}>¡Hola Gatsby!</div>
+  <div style={{ color: `purple`, fontSize: `72px` }}>Salut Gatsby!</div>
 )
 ```
 
 ❌ DON'T:
 
 ```js
-// Ejemplo
+// Exemple
 import React from "react"
 export default () => (
   // 'purple' is a CSS keyword
-  <div style={{ color: `morado`, fontSize: `72px` }}>¡Hola Gatsby!</div>
+  <div style={{ color: `morado`, fontSize: `72px` }}>Salut Gatsby!</div>
```

Comment:
In French, there is a space before the question mark

```suggestion
  <div style={{ color: `morado`, fontSize: `72px` }}>Salut Gatsby !</div>
```

Reply:
I can't manage to find your comment about "morado", so I put this here: in french we use "mauve" and "violet" as the same thing because no one agreed about what is violet and what is mauve. So you can use whatever you prefer in your translations.

Reply:
Okay, good to know.